docs/index.html

```
@@ -102,12 +112,22 @@ <h2>Installation</h2><p>Using yarn</p>
         });
 });
 
-client.connect();</code></pre><h3>Client (Browser)</h3><p>We provide client constructor as LineClient global reference. In order to run line client on browser, you must include the browser bundle into your HTML file as below and then use it by creating an instance from <em>LineClient</em> class.</p>
-<pre class="prettyprint source"><code>&lt;script src=&quot;../dist/client-web-globals.js&quot;>&lt;/script></code></pre><pre class="prettyprint source lang-js"><code>const client = new LineClient('ws://localhost:3000');
+client.connect();
+</code></pre>
+<h3>Client (Browser)</h3>
+<p>We provide client constructor as LineClient global reference. In order to run line client on browser, you must include the browser bundle into your HTML file as below and then use it by creating an instance from <em>LineClient</em> class.</p>
+<pre class="prettyprint source"><code>&lt;script src=&quot;../dist/client-web-globals.js&quot;>&lt;/script>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)